src/utils/common.ts

```diff
@@ -359,3 +360,37 @@ export function detectProtocolByOwner(network: string, address?: string) {
   }
   return null;
 }
+
+export async function defaultAccountDiscovery(
+  isAccountUsed: Function,
+  getHdWalletAccountFromMnemonicSeed: Function,
```

Comment:
Instead of `Function` I suggest to use for example: `(address: string) => Promise<boolean>`.